Mathlib/MeasureTheory/Order/Group/Lattice.lean

```diff
@@ -0,0 +1,33 @@
+/-
+Copyright (c) 2024 Xavier Roblot. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Xavier Roblot
+-/
+import Mathlib.Algebra.Order.Group.PosPart
+import Mathlib.MeasureTheory.Group.Arithmetic
+import Mathlib.MeasureTheory.Order.Lattice
+
+
+/-!
+# Measurability results on groups with a lattice structure.
+
+## Tags
+
+measurable function, group, lattice operation
+-/
+
+variable {R : Type*} [Lattice R] [Group R] [MeasurableSpace R] [MeasurableSup₂ R]
+
+@[to_additive (attr := measurability)]
+theorem measurable_oneLePart : Measurable fun x : R ↦ oneLePart x := by
+  refine Measurable.sup measurable_id' measurable_const
+
+variable [MeasurableInv R]
+
+@[to_additive (attr := measurability)]
+theorem measurable_mabs : Measurable fun x : R ↦ mabs x :=
```

Comment:
Could you also add three lemmas of that form (also with the measurability attribute):
`lemma measurable.abs {f : α → β} (hf : Measurable f) : Measurable (fun x ↦ |f x|) := sorry`

It's an immediate consequence of what you proved, but writing `hf.abs` is more convenient than `measurable_abs.comp hf`.

Reply:
There you go :)